<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frame</class>
 <widget class="QFrame" name="Frame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="akVLayout">
     <property name="topMargin">
      <number>6</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="iPoseLayout">
         <property name="spacing">
          <number>10</number>
         </property>
         <property name="leftMargin">
          <number>20</number>
         </property>
         <item>
          <widget class="QLabel" name="stp01">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>24</height>
            </size>
           </property>
           <property name="text">
            <string>Step 1:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="mGearRootName">
           <property name="minimumSize">
            <size>
             <width>160</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>160</width>
             <height>24</height>
            </size>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="placeholderText">
            <string>Select mGear Skeleton Root</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="loadMGearSkelButton">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>120</width>
             <height>24</height>
            </size>
           </property>
           <property name="text">
            <string>Load Selection</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="skelConfigLayoutStep2">
         <property name="spacing">
          <number>10</number>
         </property>
         <property name="leftMargin">
          <number>20</number>
         </property>
         <property name="rightMargin">
          <number>10</number>
         </property>
         <item>
          <widget class="QLabel" name="skelConfigLabel">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>24</height>
            </size>
           </property>
           <property name="text">
            <string>Step 2:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cExportTypeComboBox">
           <property name="placeholderText">
            <string>Choose Character Export Type</string>
           </property>
           <item>
            <property name="text">
             <string>HAnimHumanoid - BASIC</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>HAnimHumanoid - RAWKEE</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>CGE Skin (glTF-style)</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="loaLayoutStep1">
         <property name="spacing">
          <number>10</number>
         </property>
         <property name="leftMargin">
          <number>20</number>
         </property>
         <property name="rightMargin">
          <number>10</number>
         </property>
         <item>
          <widget class="QLabel" name="laoLabel">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>24</height>
            </size>
           </property>
           <property name="text">
            <string>Step 3:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="loaComboBox">
           <property name="placeholderText">
            <string>Choose your Level of Articulation (LOA)</string>
           </property>
           <item>
            <property name="text">
             <string>Level -1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Level 0</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Level 1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Level 2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Level 3</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Level 4</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="aPoseLayout">
         <property name="spacing">
          <number>10</number>
         </property>
         <property name="leftMargin">
          <number>20</number>
         </property>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>24</height>
            </size>
           </property>
           <property name="text">
            <string>Step 4:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="createRKSkelButton">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>120</width>
             <height>24</height>
            </size>
           </property>
           <property name="text">
            <string>Create RK Skeleton</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="tPoseLayout">
         <property name="spacing">
          <number>10</number>
         </property>
         <property name="leftMargin">
          <number>20</number>
         </property>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>24</height>
            </size>
           </property>
           <property name="text">
            <string>Step 5:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="minimumSize">
            <size>
             <width>260</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>260</width>
             <height>24</height>
            </size>
           </property>
           <property name="text">
            <string>Select Skin Meshes to be Transfer to RK Skeleton</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="ePoseLayout">
         <property name="spacing">
          <number>10</number>
         </property>
         <property name="leftMargin">
          <number>20</number>
         </property>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>24</height>
            </size>
           </property>
           <property name="text">
            <string>Step 6:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="transferMeshButton">
           <property name="minimumSize">
            <size>
             <width>165</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>165</width>
             <height>24</height>
            </size>
           </property>
           <property name="text">
            <string>Transfer Mesh to RK Skeleton</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="customLayout">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>20</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QLabel" name="customPoseLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Restore / Remove Customized RawKee Poses</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="comboBoxLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>10</number>
         </property>
         <property name="rightMargin">
          <number>10</number>
         </property>
         <item>
          <widget class="QComboBox" name="customBPBox">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="placeholderText">
            <string>List of RawKee Bind Poses</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="removeRestoreLayout">
         <property name="leftMargin">
          <number>40</number>
         </property>
         <property name="rightMargin">
          <number>40</number>
         </property>
         <item>
          <widget class="QPushButton" name="pushButton_4">
           <property name="text">
            <string>Remove</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_3">
           <property name="text">
            <string>Restore</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="customEditLineLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>40</number>
         </property>
         <property name="rightMargin">
          <number>40</number>
         </property>
         <item>
          <widget class="QLineEdit" name="lineEdit">
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="placeholderText">
            <string>Type a new Pose Name and Press Enter to Create Pose</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="poseInterface">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="iPoseLayout">
         <property name="spacing">
          <number>10</number>
         </property>
         <property name="leftMargin">
          <number>50</number>
         </property>
         <item>
          <widget class="QPushButton" name="restoreIPose">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>24</height>
            </size>
           </property>
           <property name="text">
            <string>Restore I-Pose</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="iPoseCheckBox">
           <property name="minimumSize">
            <size>
             <width>170</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>170</width>
             <height>24</height>
            </size>
           </property>
           <property name="text">
            <string>Is Set as mGear Pose</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="aPoseLayout">
         <property name="spacing">
          <number>10</number>
         </property>
         <property name="leftMargin">
          <number>50</number>
         </property>
         <item>
          <widget class="QPushButton" name="restoreAPose">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>24</height>
            </size>
           </property>
           <property name="text">
            <string>Restore A-Pose</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="aPoseCheckBox">
           <property name="minimumSize">
            <size>
             <width>170</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>170</width>
             <height>24</height>
            </size>
           </property>
           <property name="text">
            <string>Is Set as mGear Pose</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="tPoseLayout">
         <property name="spacing">
          <number>10</number>
         </property>
         <property name="leftMargin">
          <number>50</number>
         </property>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>24</height>
            </size>
           </property>
           <property name="text">
            <string>Restore T-Pose</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox">
           <property name="minimumSize">
            <size>
             <width>170</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>170</width>
             <height>24</height>
            </size>
           </property>
           <property name="text">
            <string>Is Set as mGear Pose</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="ePoseLayout">
         <property name="spacing">
          <number>10</number>
         </property>
         <property name="leftMargin">
          <number>50</number>
         </property>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>24</height>
            </size>
           </property>
           <property name="text">
            <string>Restore E-Pose</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_2">
           <property name="minimumSize">
            <size>
             <width>170</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>170</width>
             <height>24</height>
            </size>
           </property>
           <property name="text">
            <string>Is Set as mGear Pose</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
